src/features/auth/components/LoginDialog.tsx

```diff
@@ -55,7 +55,7 @@ export const LoginDialog: FC<LoginDialogProps> = ({ open, handleClose }) => {
                 variant="contained"
```

Comment:
up here a bit ^ ..can we have `type="password"` on the password `TextField` please?

Reply:
Will do in a new PR- also want to make 'enter' work!!